ui/analyse/src/study/studyMembers.ts

```diff
@@ -218,7 +218,7 @@ export function view(ctrl: StudyCtrl): VNode {
       .map(member => {
         const confing = members.confing() === member.user.id;
         return [
-          h('div', { key: member.user.id, class: { editing: !!confing } }, [
+          h('button', { key: member.user.id, class: { editing: !!confing } }, [
```

Comment:
why are these buttons? They don't do anything when clicked. And they contain an `<a href>`.